canvas_course_admin_tools/requirements/local.txt

```diff
@@ -4,14 +4,22 @@
  
 # below are requirements specific to the local environment
 
-django-debug-toolbar==1.3.2
-django-sslserver==0.15
-mock==1.3.0
-pep8==1.6.2
-flake8==2.4.1
+ddt==1.0.1
+django-debug-toolbar==1.4
+django-sslserver==0.19
+flake8==2.5.4
+mock==2.0.0
+oauthlib==1.1.1
+pep8==1.7.0
+pyvirtualdisplay==0.2
+requests-oauthlib==0.6.1
+selenium==2.53.2
+xlrd==0.9.4
 
 # icommons_common unit tests depend on ccsw, and ccsw depends on icommons_common.
```

Comment:
If we're not actually going to run icommons_common unit tests, perhaps we can remove this completely and note it in our tech debt ticket.